cli/pkg/kctrl/cmd/app/app_tailer.go

```diff
@@ -49,7 +49,8 @@ func (o *AppTailer) printTillCurrent(status kcv1alpha1.AppStatus) error {
 		return nil
 	}
 
-	completed, err := NewAppStatusDiff(kcv1alpha1.AppStatus{}, status, o.statusUI).PrintUpdate()
+	completed, cachedDeployOutput, err := NewAppStatusDiff(kcv1alpha1.AppStatus{}, status, o.statusUI, o.lastSeenDeployStdout).PrintUpdate()
```

Comment:
nit: rename `cachedDeployOutput` to `deployOutput`. Same everywhere

Reply:
Makes sense, because it is not cached at this point! Will make the change.

Reply:
Gonna wait for another review and do this change with other suggested changes